red-datasets.gemspec

```diff
@@ -43,4 +43,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("test-unit")
   spec.add_development_dependency("yard")
   spec.add_development_dependency("kramdown")
+  spec.add_development_dependency("steep")
+  spec.add_development_dependency("typeprof")
```

Comment:
Ah, could you use `Gemfile` for development dependencies?

(We should move all `add_development_dependency`s eventually but we should work on it as a separated PR.)

Reply:
I moved added gems to Gemfile with this commit. [022ada1](https://github.com/red-data-tools/red-datasets/pull/238/commits/022ada19106818f086248eeb40d761da81ebd78a)